src/classes/User.js

```diff
@@ -1,8 +1,9 @@
 class User {
-  constructor(fullname, position, photo) {
+  constructor(fullname, position, photo, testimonial) {

```

Comment:
`src/classes/User.js` classes it is not a good name for a directory. Better name would be: `models`.  This is often used name for directory which holds business entities